pkg/adapter/v2/cloudevents.go

```diff
@@ -32,6 +36,10 @@ import (
 	"knative.dev/pkg/tracing/propagation/tracecontextb3"
 )
 
+const (
+	EnvSinkWaitDuration = "K_SINK_WAIT_DURATION"
```

Comment:
The name of this env doesn't give a clear idea of what is about, I prefer to rename it to `K_SINK_TIMEOUT`

Reply:
Sink wait duration sounds good to me